frontend/package.json

```diff
@@ -41,6 +42,7 @@
 		"prettier": "^2",
 		"prettier-plugin-svelte": "^2",
 		"rimraf": "^4",
+		"sass": "^1.62.1",
```

Comment:
I mentioned this else where, but we're already using postcss, do we need sass?

Reply:
Thanks for that. I had no idea what postcss is. [Now I do](https://zerotomastery.io/blog/postcss-vs-sass/).